calm_adapter/calm_adapter/src/test/scala/uk/ac/wellcome/calm_adapter/CalmAdapterWorkerServiceTest.scala

```diff
@@ -43,7 +43,7 @@ class CalmAdapterWorkerServiceTest
     withCalmAdapterWorkerService(retriever, store) {
       case (calmAdapter, QueuePair(queue, dlq), topic) =>
         sendNotificationToSQS(queue, CalmWindow(queryDate))
-        Thread.sleep(500)
+        Thread.sleep(1500)
```

Comment:
If this fixes the tests thats an argument for using `eventually`, but still prefer this way :see_no_evil:

Reply:
I'm not sure if it does.. I only see the failure in travis. I can't replicate it in any way locally it seems

Reply:
RE comment above regarding use of `eventually`, I generally prefer eventually in these cases because you can set a very long timeout for cases where things may be slow in CI, but have it run faster if things are up and responding. `Thread.sleep` always forces you to wait, while `eventually doesn't